graphblas_algorithms/interface.py

```diff
@@ -191,6 +193,10 @@ def convert_from_nx(
             raise NotImplementedError("non-None `node_attrs` is not implemented")
         if preserve_node_attrs:
             raise NotImplementedError("`preserve_node_attrs=True` is not implemented")
+        if preserve_graph_attrs:
+            raise NotImplementedError("`preserve_graphs_attrs=True` is not implemented")
+        if graph_name:
+            raise NotImplementedError("Not possible to set a graph name")
```

Comment:
btw, this `graph_name` corresponds to the _argument name_ in the networkx algorithm, which can be useful for debugging and logging. Also, `graphblas_algorithms` Graphs also support setting the name. Nevertheless, I'll merge this just to add you as a contributor ;)